major_event_log/templates/major-event-log/about.html

```diff
@@ -8,6 +8,6 @@
 		<h1>About</h1>
 	</div>
     <p>This application serves to document major events related to the UNT Library's digital preservation projects. These events are not tied to specific objects, but rather affect all preserved objects or major subsets of objects.</p>
-    <p>The events themselves can be viewed collectively on the home page or viewed individually by clicking on the appropriate event's title. When viewed individually, the event is displayed in plain HTML. If desired, the event can also be displayed using the PREMIS format by clicking on either of the two blue buttons underneath the individual event's details. The 'PREMIS XML' button will show <b>only</b> the PREMIS XML, while the 'Atom' button will present the PREMIS XML embedded in an Atom object. Additionally, an Atom feed is available by clicking the 'Atom Feed' link on the right of the navigation bar, which will always show the 10 most recent events, provided they exist.</p>
+    <p>The events themselves can be viewed collectively on the home page or viewed individually by clicking on the appropriate event's ID. When viewed individually, the event is displayed in plain HTML. If desired, the event can also be displayed in XML by clicking on either of the two blue buttons underneath the individual event's details. The 'PREMIS' button will show <b>only</b> the PREMIS XML, while the 'Atom' button will present the PREMIS XML embedded in an Atom object. Additionally, an Atom feed is available by clicking the 'Atom Feed' link on the right of the navigation bar, which will always show the 10 most recent events, provided they exist.</p>
```

Comment:
I could see a lot of this content being cut out to make this more concise. If you want to name formats, we could do it more briefly just saying the events are available in HTML, PREMIS XML, and Atom formats. Also, we can make the Atom feed info a bit shorter.

Also, if we are going to specifically name UNT Libraries on this page, it should be written as plural UNT Libraries, not UNT Library.